docroot/themes/humsci/humsci_basic/src/scss/components/_masthead.scss

```diff
@@ -5,14 +5,31 @@
 
   .su-lockup {
     z-index: $hb-z-index-lockup;
+
```

Comment:
Perhaps my lockup isn't setup correctly but I'm getting some funky spacing between the text. Is this expected?
<img width="462" alt="Screen Shot 2020-03-23 at 7 29 25 AM" src="https://user-images.githubusercontent.com/2486846/77312254-0d4b5d00-6cd8-11ea-9432-8be1a798370c.png">